multimodal/src/autogluon/multimodal/learners/matching.py

```diff
@@ -2013,52 +2011,37 @@ def _load_metadata(
         with open(os.path.join(path, "assets.json"), "r") as fp:
             assets = json.load(fp)
 
-        query_config = upgrade_config(query_config, assets["version"])
-        response_config = upgrade_config(response_config, assets["version"])
-
         with open(os.path.join(path, "df_preprocessor.pkl"), "rb") as fp:
-            df_preprocessor = CustomUnpickler(fp).load()
+            df_preprocessor = pickle.load(fp)  # nosec B301
```

Comment:
What is CustomUnpickler and why did we stop using it in many areas of the code?

Reply:
CustomUnpickler is a wrapper on top of pickle. It was a legacy used to load models saved when automm was still within autogluon.text. Since we autogluon.text was deprecated a long time ago, we no longer need to support it.